src/test/java/core/basesyntax/model/OperationTest.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class OperationTest {
+    private static Operation actual;
+    private static Operation expected;
+
+    @Test
+    public void getOperationByShortName_balance_Ok() {
+        actual = Operation.getOperationByShortName("b");
+        expected = Operation.BALANCE;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void getOperationByShortName_purchase_Ok() {
+        actual = Operation.getOperationByShortName("p");
+        expected = Operation.PURCHASE;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void getOperationByShortName_supply_Ok() {
+        actual = Operation.getOperationByShortName("s");
+        expected = Operation.SUPPLY;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void getOperationByShortName_return_Ok() {
+        actual = Operation.getOperationByShortName("r");
+        expected = Operation.RETURN;
+        assertEquals(expected, actual);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void getOperationByShortName_incorrectOperation_NotOk() {
+        Operation.getOperationByShortName("m");
```

Comment:
declare m as a variable with informative name, now it's not clear why test works with 'r' but not with 'm'